daml-lf/transaction/src/test/scala/com/digitalasset/daml/lf/transaction/TransactionCoderSpec.scala

```diff
@@ -173,7 +173,8 @@ class TransactionCoderSpec
         }
       }
 
-    "succeed with encoding under later version if succeeded under earlier version" in
+    // FIXME: do not accept a PR that contains this comment
```

Comment:
Well, you won't get my approval then.

Reply:
I have to fix the test first.

Reply:
The test is fixed. The comment is dropped.